ServantPersistent/src/Types.hs

```diff
@@ -0,0 +1,124 @@
+{-# LANGUAGE DataKinds       #-}
+{-# LANGUAGE KindSignatures       #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeOperators   #-}
+{-# LANGUAGE DeriveGeneric   #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE DeriveAnyClass  #-}
+{-# LANGUAGE DeriveFunctor  #-}
+{-# LANGUAGE TypeInType  #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE FunctionalDependencies #-}
+{-# LANGUAGE PolyKinds  #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE GADTs  #-}
```

Comment:
If there are used across all your Haskell files, you can put them in cabal, I think.